oshdb-tool/etl/src/main/java/org/heigit/ohsome/oshdb/tool/importer/util/SizeEstimator.java

```diff
@@ -1,18 +1,18 @@
 package org.heigit.ohsome.oshdb.tool.importer.util;
 
 public class SizeEstimator {
-  
-  private static int OBJ_HEADER;
-  private static int ARR_HEADER;
-  private static int PADDING;
-  private static int INT_FIELD = 4;
-  private static int OBJ_REF;
-  private static int OBJ_OVERHEAD;
+
+  private static long OBJ_HEADER;
+  private static long ARR_HEADER;
+  private static long PADDING;
+  private static long INT_FIELD = 4;
+  private static long OBJ_REF;
+  private static long OBJ_OVERHEAD;
```

Comment:
why was this changed? I can't see at first glance why this would be a "code style" fix :thinking:

Reply:
Yes this is not a code-style but a sonarlint fix.  Basically the complain was that at least one compontent of a summation which should return a long value have to be a long value as well.